Resource.js

```diff
@@ -80,10 +82,13 @@ module.exports = function(app, route, modelName, model) {
                 return _.pick(error, 'path', 'name', 'message');
               })
             });
-          case 404:
```

Comment:
Why did you remove the `404` response? This will break all the 404 responses for content not found.